lib/workers/repository/update/pr/pr-fingerprint.ts

```diff
@@ -0,0 +1,99 @@
+// fingerprint config is based on the old skip pr update logic
+// https://github.com/renovatebot/renovate/blob/3d85b6048d6a8c57887b64ed4929e2e02ea41aa0/lib/workers/repository/update/pr/index.ts#L294-L306
+
+import type { UpdateType, ValidationMessage } from '../../../../config/types';
+import { pkg } from '../../../../expose.cjs';
+import { logger } from '../../../../logger';
+import type { PrCache } from '../../../../util/cache/repository/types';
+import { getElapsedHours } from '../../../../util/date';
+import type { BranchConfig } from '../../../types';
+
+// BranchUpgradeConfig - filtered
+interface FilteredBranchUpgradeConfig {
```

Comment:
```suggestion
export interface FilteredBranchUpgradeConfig {
```
?

Reply:
Its not used anywhere else though!

Reply:
it should be exported because the parent interface is exported. so never have public interfaces where some properties are using private types.

Reply:
Got it! Thanks.